### ISSUE TITLE
🐛[BUG] [System/Item] Return/Homing ring issue

### ISSUE THREAD

**Issue by captOFP** [HorizonFFXI/HorizonXI-Issues#676]
**What does this bug affect?**
 Return ring usage, does not work, ignores action when trying to use and does not transport to regional outpost.

**Steps to Reproduce**
 1. be in outside area of a region controlled by home nation. 2. have completed the outpost supplies quest for that region. 3. be of appropriate level. 4. use return/homing ring to be transported to the outpost.

**Expected behavior**
 step 4 above should occur.

**Screenshots**
 NA

**Setup Information**
 current.

**Additional context**
 NA

**Comment by CriticalXI** [comment]
Can you please confirm what outpost you tried to use this ring to teleport to and your character level?

**Comment by captOFP** [comment]
I was on on 23 thf, and tried using the ring in meriphataud mountains, sauromugue champaign, and pashhhow marshlands.  I was just getting a message to the effect of, "cannot be used in this area."

**Comment by dallano** [comment]
I agree. in your steps to reproduce you state that you're in a region where another nation owns the outpost. I also believe it functions when your nation owns the op
